rust/agents/relayer/src/msg/pending_message.rs

```diff
@@ -359,8 +415,8 @@ impl PendingMessage {
 
 #[derive(Debug)]
 pub struct MessageSubmissionMetrics {
-    last_known_nonce: IntGauge,
-    messages_processed: IntCounter,
+    pub last_known_nonce: IntGauge,
```

Comment:
This change to `pub` fields feels odd.

Reply:
I needed the fields to be public so I could construct a dummy instance in the unit tests: https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/72e13bc8deaec05d218916bbdc91e6db92ce6445/rust/agents/relayer/src/msg/processor.rs#L240